lib/jsonapi_compliable/adapters/null.rb

```diff
@@ -4,8 +4,179 @@ module Adapters
     # Useful when your customization does not support all possible
     # configuration (e.g. the service you hit does not support sorting)
     class Null < Abstract
-      # (see Adapters::Abstract#filter)
-      def filter(scope, attribute, value)
+      def filter_string_eq(scope, attribute, value)
```

Comment:
Lots of repeated methods here. Would prefer a single filter_noop (or whatever) with a bunch of aliases.

Reply:
Nothing is stopping an implementor from doing so, in fact I do so in the PORO adapter. The purpose of these spelled-out methods is to provide a baseline of common operations so it's pretty obvious how to subclass and overrides. We could do the same with aliases, but I don't think we gain much, and we would be adding a (small) mental hop for those following the code.